Lib/tracemalloc.py

```diff
@@ -182,15 +182,16 @@ class Traceback(Sequence):
     Sequence of Frame instances sorted from the oldest frame
     to the most recent frame.
     """
-    __slots__ = ("_frames",)
+    __slots__ = ("_frames", 'nframe')
 
-    def __init__(self, frames):
+    def __init__(self, frames, nframe):
         Sequence.__init__(self)
         # frames is a tuple of frame tuples: see Frame constructor for the
         # format of a frame tuple; it is reversed, because _tracemalloc
         # returns frames sorted from most recent to oldest, but the
         # Python API expects oldest to most recent
         self._frames = tuple(reversed(frames))
+        self.nframe = nframe
```

Comment:
I would prefer to make it private and add a read-only property. The API is designed to make all data read-only to ensure that all data remain consistent.

Reply:
Sure!